### PR TITLE
Issue/mode test

### PR DESCRIPTION
**Pull request by wouterdb**
# Description

To be reviewed after the merge of https://github.com/inmanta/inmanta-core/pull/3271

Actually enable fast model

This makes the following tests nightly

- older DB migrations
- project freeze
- test_compile_runner
- test_project_install_modules_cache_invalid
- test_module_update_with_v2_module

This is a 33% runtime reduction, 1% coverage reduction (89% instead of 90%)

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -15,6 +15,22 @@
 
     Contact: code@inmanta.com
 """
+
+"""
+About the use of @parametrize_any and @slowtest:
```
Comment (arnaudsjs) [inline tests/conftest.py:20]: Cool. Now it's very clear how these annotations should be used.

**Review by arnaudsjs — APPROVED** [review]

File: tests/conftest.py [hunk 2]
```diff
@@ -15,6 +15,22 @@
...
+
+For parametrized tests:
+- if the test is fast and tests different things for each parameter, use @parametrize
+- if the parameters only slightly increase the coverage of the test (some different exception path,...), use @parametrize_any.
+- if a test is slow, use @parametrize_any
```
Comment (sanderr) [inline tests/conftest.py:25]: If a test is slow, even if the parameters significantly increase the coverage?

File: tests/test_env.py [hunk 3]
```diff
@@ -229,9 +229,9 @@ def test_process_env_install_from_index_conflicting_reqs(
     assert package_name not in env.process_env.get_installed_packages()
 
 
-@pytest.mark.parametrize("editable", [True, False])
+@pytest.mark.parametrize_any("editable", [True, False])
```
Comment (sanderr) [inline tests/test_env.py:232]: This one tests fundamentally different behavior in my opinion. I would leave this as just `parametrize`. (Only this parameter, the other increases the test coverage only by a little.)

**Review by sanderr — APPROVED** [review]
Two minor comments, nothing I feel very strongly about.

File: tests/conftest.py (hunk 2 above)
Comment (wouterdb) [reply tests/conftest.py:25]: yes, but this is a tradeoff (as both slow and coverage are vague things)

File: tests/test_env.py (hunk 3 above)
Comment (wouterdb) [reply tests/test_env.py:232]:
```suggestion
@pytest.mark.parametrize("editable", [True, False])
```

**Pull request closed without merging**